srv/SetIO.srv

```diff
@@ -16,6 +16,27 @@
 #  state  Desired tool voltage (use STATE_TOOL_VOLTAGE constants)
 
 # constants
+# pin mapping
+int8 DOUT0 = 0
+int8 DOUT1 = 1
+int8 DOUT2 = 2
+int8 DOUT3 = 3
+int8 DOUT4 = 4
+int8 DOUT5 = 5
+int8 DOUT6 = 6
+int8 DOUT7 = 7
+int8 COUT0 = 8
+int8 COUT1 = 9
+int8 COUT2 = 10
+int8 COUT3 = 11
+int8 COUT4 = 12
+int8 COUT5 = 13
+int8 COUT6 = 14
+int8 COUT7 = 15
+int8 TOOL_OUT0 = 16
+int8 TOOL_OUT1 = 17
+int8 AOUT0 = 0
+int8 AOUT1 = 1
```

Comment:
let's be very clear about what pins these constants refer to -- it will only save us from answering questions later.

I've also prefixed them all with `PIN_`, both to align with the other constants (`FUN_..`) and to avoid potential clashes in the future.

```suggestion
# pin mapping
# analog out
int8 PIN_AOUT0 = 0
int8 PIN_AOUT1 = 1

# digital out
int8 PIN_DOUT0 = 0
int8 PIN_DOUT1 = 1
int8 PIN_DOUT2 = 2
int8 PIN_DOUT3 = 3
int8 PIN_DOUT4 = 4
int8 PIN_DOUT5 = 5
int8 PIN_DOUT6 = 6
int8 PIN_DOUT7 = 7

# configurable out
int8 PIN_CONF_OUT0 = 8
int8 PIN_CONF_OUT1 = 9
int8 PIN_CONF_OUT2 = 10
int8 PIN_CONF_OUT3 = 11
int8 PIN_CONF_OUT4 = 12
int8 PIN_CONF_OUT5 = 13
int8 PIN_CONF_OUT6 = 14
int8 PIN_CONF_OUT7 = 15

# digital tool output
int8 PIN_TOOL_DOUT0 = 16
int8 PIN_TOOL_DOUT1 = 17
```